edb/lib/std/60-baseobject.edgeql

```diff
@@ -44,7 +44,7 @@ CREATE ABSTRACT TYPE std::Object EXTENDING std::BaseObject {
         'Root object type for user-defined types';
 };
 
-CREATE TYPE std::FreeObject EXTENDING std::BaseObject {
+CREATE ABSTRACT TYPE std::FreeObject EXTENDING std::BaseObject {
```

Comment:
Hmm, technically `abstract` means that there can be no instance of this type _exactly_, so this qualifier would be a lie.  I think we should instead prohibit `insert FreeObject` explicitly and leave the type non-abstract.

Reply:
Yeah, fair